lib/src/cmaf/ham/mapper/mpd/utilsHamToMpd.ts

```diff
@@ -0,0 +1,32 @@
+import type { AudioTrack, Track } from '../../types/model';
+
+/**
+ * This function tries to recreate the timescale value.
+ *
+ * This value is not stored on the ham object, so it is not possible (for now)
+ * to get the original one.
+ *
+ * Just the audio tracks have this value stored on the `sampleRate` key.
+ *
+ * Using 90000 as default for video since it is divisible by 24, 25 and 30
+ *
+ * Using 1000 as default for text
+ *
+ * @param track Track to get the timescale from
+ * @returns Timescale in numbers
+ */
+function getTimescale(track: Track): number {
+	if (track.type === 'audio') {
+		const audioTrack = track as AudioTrack;
+		return audioTrack.sampleRate;
+	}
+	if (track.type === 'video') {
+		return 90000;
```

Comment:
Can we add them as constansts?

Reply:
Agreed. Move `90000`, `1000` and `48000` as constants at the top of the file and move the comment to each constant.